superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/getControlItemsMap.tsx

```diff
@@ -149,6 +150,7 @@ export default function getControlItemsMap({
       (controlItem: CustomControlItem) =>
         controlItem?.config?.renderTrigger &&
         controlItem.name !== 'sortAscending' &&
+        controlItem?.config?.type !== 'SelectControl' &&
```

Comment:
Why a checkbox? I added a whole new section (below) to handle dropdown selects because that doesn't exist anywhere else in the config modal. It's for the step size and scaling function select boxes

Reply:
Oh I reread this and get it now. I guess the question is, how flexible should these controls be? And since everything else is a checkbox, no one bothered to check before. I'll make that change, thanks!